aries-site/src/examples/components/pageheader/utils.js

```diff
@@ -2,7 +2,7 @@ export const groupActions = (actions, breakpoint, bestPractice) => {
   let collapsedActions;
   let displayedActions;
   // collapse all default/secondary actions
-  if (breakpoint === 'medium') {
+  if (['medium', 'large'].includes(breakpoint)) {
```

Comment:
My expectation is that action items will only collapse into the overflow menu [when space runs out](https://design-system.hpe.design/components/pageheader#responsive-behavior-and-content-prioritization). This is collapsing when plenty of space is still available:

![Screen Shot 2022-10-25 at 1 41 18 PM](https://user-images.githubusercontent.com/1756948/197868285-8fd48b07-ed49-405e-a944-340828cddf15.png)

I would expect that the actions area probably has unique definitions across breakpoints.

Reply:
right here if all 3 buttons were shown they would wrap because the fact we have the anatomy grid laid over the PageHeader grid the breakpoints arent working smoothly so Im going to try and build it all into one grid.